### PR TITLE
ISD-2288: Update tutorial

### PR DESCRIPTION
**Pull request by alithethird**
Applicable spec: ISD-2288 (internal)

### Overview

Updated the tutorial to standardize.
### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -11,26 +11,37 @@ also helps operate the charm by liaising with the Canonical Observability Stack
...
 juju add-model wordpress-tutorial
 ```
 
-### Deploy wordpress-k8s charm
+### Deploy Wordpress K8s charm
 
 Deployment of WordPress requires a relational database. The integration with the
 `mysql` [interface](https://juju.is/docs/sdk/integration) is required by the wordpress-k8s
-charm and hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.
+charm and hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.For more information, see [Charm Architecture](https://charmhub.io/indico/docs/explanation-charm-architecture).
```
Comment (erinecon) [inline docs/tutorial.md:44]: Why does this tutorial link to the Indico docs?

Comment (alithethird) [reply docs/tutorial.md:44]: Oh, sorry! Nice catch!

File: docs/tutorial.md [hunk 2]
```diff
@@ -91,8 +100,21 @@ password: <password> # should look something like: XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 ```
 
-You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with
-username admin and password the action above.
+> NOTE: If you are using Multipass VM for this tutorial, you will need to route the IP from Multipass. To do this first get the IP of he Multipass VM.
```
Comment (erinecon) [inline docs/tutorial.md:103]: he --> the, in other words:

> To do this first get the IP of **the** Multipass VM.

File: docs/tutorial.md [hunk 3]
```diff
@@ -91,8 +100,21 @@ password: <password> # should look something like: XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
...
+> ```
+> The IP you see here will be called <VM_IP> in this example.
+>
+> Then route:
+> ```
+> sudo ip route add <UNIT_IP> via <VM_IP>
+> ```
+
+
+You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with username admin and password the action above.
```
Comment (erinecon) [inline docs/tutorial.md:116]: Suggested rewrite:

> You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with the admin username and password from the previous action.

File: docs/tutorial.md [hunk 4]
```diff
@@ -11,22 +11,33 @@ also helps operate the charm by liaising with the Canonical Observability Stack
...
+- Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by using a Multipass VM as outlined in this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
 
-To deploy wordpress-k8s charm, you will need a Juju bootstrapped with any Kubernetes controller.
-To see how to bootstrap your Juju installation with MicroK8s, please refer to the documentation
-on MicroK8s [installation](https://juju.is/docs/olm/microk8s).
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
-## Steps
-### Set up the tutorial model
+:warning: When using a Multipass VM, make sure to replace `127.0.0.1` IP addresses with the
```
Comment (erinecon) [inline docs/tutorial.md:19]: Is there a 127.0.0.1 IP address used in this tutorial? This warning might not be necessary for this tutorial?

File: docs/tutorial.md [hunk 5]
```diff
@@ -102,3 +124,9 @@ model environment that you’ve created using the following command.
 ```
 juju destroy-model wordpress-tutorial --destroy-storage
 ```
+
+If you used Multipass, to remove the Multipass instance you created for this tutorial, use the following command.
```
Comment (erinecon) [inline docs/tutorial.md:128]: Nitpick: We may want to tell the reader to first `exit` from the VM before removing the Multipass instance.

File: docs/tutorial.md [hunk 6]
```diff
@@ -91,8 +100,21 @@ password: <password> # should look something like: XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
...
+> ```
+> The IP you see here will be called <VM_IP> in this example.
+>
+> Then route:
+> ```
+> sudo ip route add <UNIT_IP> via <VM_IP>
+> ```
+
+
+You can now access your WordPress application at `http://<UNIT_IP>/wp-login.php` and login with the admin username and password from the previous action.
```
Comment (erinecon) [inline docs/tutorial.md:116]: login --> log in
(login is a noun or adjective, while "log in" is a verb. I have to look this up constantly!)

**Review by erinecon — APPROVED** [review]
Thanks for your work on the tutorial!

**Review by javierdelapuente — APPROVED** [review]

**Review by javierdelapuente — APPROVED** [review]

**Pull request merged**